sumologic/sumologic_field.go

```diff
@@ -94,6 +94,30 @@ func (s *Client) FindFieldId(name string) (string, error) {
 	return "", fmt.Errorf("Field \"%s\" not found", name)
 }
 
+func (s *Client) DisableField(id string) error {
+	urlWithoutParams := "v1/fields/%s/disable"
+	sprintfArgs := []interface{}{}
+	sprintfArgs = append(sprintfArgs, id)
+
+	urlWithParams := fmt.Sprintf(urlWithoutParams, sprintfArgs...)
+
+	_, err := s.Delete(urlWithParams)
+
+	return err
+}
+
+func (s *Client) EnableField(id string) error {
+	urlWithoutParams := "v1/fields/%s/enable"
+	sprintfArgs := []interface{}{}
+	sprintfArgs = append(sprintfArgs, id)
+
+	urlWithParams := fmt.Sprintf(urlWithoutParams, sprintfArgs...)
+
+	_, err := s.Put(urlWithParams, nil)
+
+	return err
+}
```

Comment:
This can be simplified 
```suggestion
func (s *Client) DisableField(id string) error {
	urlWithParams := fmt.Sprintf("v1/fields/%s/disable", id)

	_, err := s.Delete(urlWithParams)
	return err
}

func (s *Client) EnableField(id string) error {
	urlWithParams := fmt.Sprintf("v1/fields/%s/enable", id)

	_, err := s.Put(urlWithParams, nil)
	return err
}

```